composer.json

```diff
@@ -26,8 +26,8 @@
     "symfony/console": "~2.8|~3.0",
     "sensio/framework-extra-bundle": "~3.0"
   },
-  "suggest": {
-    "symfony/routing": "To use the @Route annotation provided by the bundle."
+  "conflict": {
+    "symfony/framework-bundle": ">2.0,<2.8.5|<3.0.5"
```

Comment:
Doesn't ``<3.0.5`` cover ``~2.0``? 
I think this should be the opposite :
``<2.8.5|>3.0,<3.0.5``